packages/backend-common/src/service/lib/ServiceBuilderImpl.ts

```diff
@@ -121,20 +145,91 @@ export class ServiceBuilderImpl implements ServiceBuilder {
         reject(e);
       });
 
-      const server = stoppable(
-        app.listen(port, host, () => {
-          logger.info(`Listening on ${host}:${port}`);
-        }),
-        0,
-      );
+      let server: http.Server;
+
+      if (httpsSettings) {
+        logger.info('Initializing https server');
+
+        const credentials: { key: string; cert: string } = {
+          key: '',
+          cert: '',
+        };
+        const signingOptions: any = httpsSettings?.certificate;
```

Comment:
Do not use `any` here. If I see things correctly, you already have types for this.

Reply:
I am using it as a way to workaround the lack of type checking in Typescript. As such "instanceof" only works with classes, which means it wont work with Types and Interfaces. We could implement a custom type guard (method) to wrap the logic or simply cast to "any" to save code and avoid TS complaining about logic that is legal in a JS context, but doesnt work in a static type checking context.

Reply:
I wont fix the any. It will be to much work for something that is a constant and scoped to the method. :)